tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/contact/OpenPaasContactsConsumer.java

```diff
@@ -67,45 +74,51 @@ public OpenPaasContactsConsumer(@Named(OPENPAAS_INJECTION_KEY) ReactorRabbitMQCh
     }
 
     public void start() {
+        startExchange(EXCHANGE_NAME_ADD, QUEUE_NAME_ADD, DEAD_LETTER_ADD);
+        startExchange(EXCHANGE_NAME_UPDATE, QUEUE_NAME_UPDATE, DEAD_LETTER_UPDATE);
+        startExchange(EXCHANGE_NAME_DELETE, QUEUE_NAME_DELETE, DEAD_LETTER_DELETE);
+    }
+
+    public void startExchange(String exchange, String queue, String deadLetter) {
         Flux.concat(
-                sender.declareExchange(ExchangeSpecification.exchange(EXCHANGE_NAME)
+                sender.declareExchange(ExchangeSpecification.exchange(exchange)
                     .durable(DURABLE).type(BuiltinExchangeType.FANOUT.getType())),
```

Comment:
Are we certain FANOUT is the exchange for all exchanges?

Reply:
Yes that's reasonnable